setup.py

```
@@ -1,17 +1,19 @@
 from setuptools import setup, find_packages
 
-with open("README.md", "r", encoding="utf-8") as fh:
+from pathlib import Path
+readme_path = Path(__file__).parent / "README.md"
+with open(readme_path, "r", encoding="utf-8") as fh:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)